tests/codegen/box-uninit-bytes.rs

```diff
@@ -8,7 +8,7 @@ use std::mem::MaybeUninit;
 pub fn box_uninitialized() -> Box<MaybeUninit<usize>> {
     // CHECK-LABEL: @box_uninitialized
     // CHECK-NOT: store
-    // CHECK-NOT: alloca
+    // CHECK-NOT: "alloca "
```

Comment:
This would literally look for `"alloca "`. What you probably want here is something like this:
```suggestion
    // CHECK-NOT: alloca{{ }}
```